roles/ovirt-ova-pack/files/pack_ova.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python
+
+import os
+import io
+import tarfile
+import sys
+import time
+
+BLOCKSIZE = 512
+NUL = "\0"
+buf = bytearray(4096)
+
+def createTarInfo(name, size):
+    info = tarfile.TarInfo(name)
+    info.size = size
+    info.mtime = time.time()
+    return info
+
+def padToBlockSize(file, size):
+    remainder = size % BLOCKSIZE
+    if remainder > 0:
+        file.write(NUL * (BLOCKSIZE - remainder))
+
+ova_path = sys.argv[1]
+print "opening for write: %s" % ova_path
+ova_fd = os.open(ova_path, os.O_WRONLY)
+ova_file = io.FileIO(ova_fd, "w")
```

Comment:
What's the use case when the target file exists?

Reply:
that is intended for quick implementation for download, i.e., the user is able to download the ova to any destination by leveraging download-image. in the quick implementation, we create the target image in one of the storage domains in advance, and the script is supposed to dump the ova's content to that image which will then be downloaded